content/SCALE/Storage/Pools/EncryptionSCALE.md

```diff
@@ -41,85 +40,207 @@ These algorithms provide authenticated encryption with block ciphers.
 
 ## Encrypting a New Dataset
 
-Users can encrypt new datasets within an existing storage pool without having to encrypt the entire storage pool.
-To encrypt a single dataset, go to **Storage**, open the <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> next to an existing dataset, and click *Add Dataset*.
+You can create new datasets within an existing storage pool as either encrypted or non-encrpted. 
+A mix of encrypted and non-encrypted datasets can exist in a single storage pool.
+ 
+To encrypt a dataset, [create a new dataset]({{< relref "/SCALE/Storage/Pools/datasetsscale.md" >}}) and after typing a name scroll down to the **Encryption Options** section.
+Encryption options on the **Add Dataset** configuration page change with the selections made.
+
+{{< tabs "Dataset Encryption Options" >}}
+{{< tab "Inherit Checkbox" >}}
+
+Child datasets inherit settings from the parent dataset.
+The **Add Dataset** configuration screen displays with the inherit checkbox already check-marked. 
+The child dataset encryption settings match the parent encryption settings.
+
+**Inherit (encrypted)** displays if the parent dataset is created with **Encryption** check-marked.
+
+![AddDatasetInheritNonEncrypted](/images/SCALE/AddDatasetInheritNonEncrypted.png "Dataset Inherit Non-Encryption")
+
+**Inherit (non-encrypted)** displays if the parent dataset is created with **Encryption** unchecked.
+
+![AddDatasetInheritEncrypted](/images/SCALE/AddDatasetInheritEncrypted.png "Dataset Inherit Encryption")
+
+You can change inherited encrypted/non-encrypted by unchecking the inherit box. 
+
+{{< /tab >}}
+{{< tab "Encryption Checkbox" >}}
+
+Click on the **Inherit (*encryption-state*)** checkbox with the checkmark to turn off the inherited encryption state. 
```

Comment:
Change sentence to
Click on the **Inherit (*encryption-state*)"" checkbox with the checkmark to turn off inherited encryption settings.